The Nokia N-Gage N80 was taken from Google and the Sony M600i added while
compatibility testing on my desk.

  - Rene Rebe <rene@exactcode.de>

--- ./drivers/usb/storage/unusual_devs.h.vanilla	2006-08-28 21:24:34.000000000 +0200
+++ ./drivers/usb/storage/unusual_devs.h	2006-08-28 21:34:13.000000000 +0200
@@ -133,6 +133,20 @@
 		US_SC_DEVICE, US_PR_DEVICE, NULL,
 		US_FL_IGNORE_RESIDUE ),
 
+/* Reported by Jiri Slaby <jirislaby@xxxxxxxxx> */
+UNUSUAL_DEV(  0x0421, 0x0446, 0x0100, 0x0100,
+		"Nokia",
+		"N80",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_IGNORE_RESIDUE | US_FL_FIX_CAPACITY ),
+
+/* Reported by Rene Rebe <rene@exactcode.de> */
+UNUSUAL_DEV(  0x0fce, 0xe031, 0x0000, 0xffff,
+		"Sony Ericsson Mobile Communications",
+		"M600i",
+		US_SC_DEVICE, US_PR_DEVICE, NULL,
+		US_FL_IGNORE_RESIDUE | US_FL_FIX_CAPACITY),
+
 /* Reported by Olaf Hering <olh@suse.de> from novell bug #105878 */
 UNUSUAL_DEV(  0x0424, 0x0fdc, 0x0210, 0x0210,
 		"SMSC",
